### ISSUE TITLE
OpenBLAS 0.3.0 test fail on ppc64le

### ISSUE THREAD

**Issue by susilehtola** [xianyi/OpenBLAS#1573]
```
./openblas_utest
TEST 1/22 amax:samax [OK]
TEST 2/22 drotmg:rotmg [OK]
TEST 3/22 drotmg:rotmg_issue1452 [OK]
TEST 4/22 drotmg:rotmg_D1eqD2_X1eqX2 [OK]
TEST 5/22 drotmg:drotmg_D1_big_D2_big_flag_zero [OK]
TEST 6/22 axpy:daxpy_inc_0 [OK]
TEST 7/22 axpy:zaxpy_inc_0 [OK]
TEST 8/22 axpy:saxpy_inc_0 [OK]
TEST 9/22 axpy:caxpy_inc_0 [OK]
TEST 10/22 zdotu:zdotu_n_1 [OK]
TEST 11/22 zdotu:zdotu_offset_1 [OK]
TEST 12/22 dsdot:dsdot_n_1 [OK]
TEST 13/22 swap:dswap_inc_0 [OK]
TEST 14/22 swap:zswap_inc_0 [OK]
TEST 15/22 swap:sswap_inc_0 [OK]
TEST 16/22 swap:cswap_inc_0 [OK]
TEST 17/22 rot:drot_inc_0 [OK]
TEST 18/22 rot:zdrot_inc_0 [OK]
TEST 19/22 rot:srot_inc_0 [OK]
TEST 20/22 rot:csrot_inc_0 [FAIL]
  ERR: test_rot.c:109  expected -2.148e-01, got 3.125e-01 (diff -5.273e-01, tol 1.000e-04)
TEST 21/22 potrf:bug_695 [OK]
TEST 22/22 potrf:smoketest_trivial [OK]
RESULTS: 22 tests (21 ok, 1 failed, 0 skipped) ran in 0 ms
make[1]: *** [Makefile:35: run_test] Error 1
make: *** [Makefile:114: tests] Error 2
make[1]: Leaving directory '/builddir/build/BUILD/openblas-0.3.0/Rblas/utest'
make: Leaving directory '/builddir/build/BUILD/openblas-0.3.0/Rblas'
error: Bad exit status from /var/tmp/rpm-tmp.XXwtRl (%build)
```
https://kojipkgs.fedoraproject.org//work/tasks/4862/27154862/build.log

**Comment by martin-frbg** [comment]
This is #1571 , fixed by PR #1572

**Comment by susilehtola** [comment]
Thanks, that fixed it!
